FileManager_OOP/FileManager_OOP/Command/CopyCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using FileManager.Commands.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FileManager.Commands
+{
+    internal class CopyCommand : FileManagerCommand
+    {
+        private readonly IUserInterface _UserInterface;
+        private readonly FileManagerLogic _FileManager;
+        public override string Description => "To copy files";
+
+        public CopyCommand(IUserInterface UserInterface, FileManagerLogic FileManager)
+        {
+            _UserInterface = UserInterface;
+            _FileManager = FileManager;
+        }
+
+        static void CopyDir(string FromDir, string ToDir)
+        {
+            Directory.CreateDirectory(ToDir);
+            foreach (string first in Directory.EnumerateFiles(FromDir))
+            {
+                var second = Path.Combine(ToDir, Path.GetFileName(first));
+                File.Copy(first, second);
+            }
+            foreach (string s in Directory.EnumerateDirectories(FromDir))
+            {
+                CopyDir(s, ToDir + "\\" + Path.GetFileName(s));
```

Comment:
```suggestion
                CopyDir(s, Path.Combine(ToDir, Path.GetFileName(s)));
```
Path.Combine - безопасный способ объединить части пути.